static/src/javascripts/projects/common/views/svgs.js

```diff
@@ -1,5 +1,10 @@
 // @flow
-// Include any images needed in templates here.
+
+// DEPRECATED: please don't use this file to inject SVGs into your markup
```

Comment:
Proper good shout 👍  I've added a note on how not to break the tests! We can fully document the new process when we remove this module entirely.